PlusServerLauncher/PlusServerLauncherMainWindow.cxx

```diff
@@ -890,3 +963,169 @@ void PlusServerLauncherMainWindow::OnTimerTimeout()
     m_RemoteControlServerConnector->PeriodicProcess();
   }
 }
+
+//----------------------------------------------------------------------------
+void PlusServerLauncherMainWindow::SendServerStartedCommand(const char* configFilePath)

```

Comment:
Avoid the use of char*, use std::string instead

Reply:
The send server started and stopped should be put on an observer mechanism, where clients can opt-in to these status messages.